docs/book/v3/getting-started/quick-start.md

```diff
@@ -590,15 +590,16 @@ The above looks for a query string parameter "target", and uses its value to
 provide to the template, which is then rendered and returned in an HTML
 response.
 
-Now, let's edit the template file to have the following contents:
+Now, let's edit the template file to have the one of the following header lines (use the one for your chosen template renderer):
 
-```php
-<h1>Hello <?= $this->target ?></h1>
-```
+```html
```

Comment:
PHP was correct, because the interesting part of this code sample is PHP and not HTML.

Reply:
Changed it to php but that doesn't seem to render nice when previewing the markdown file.

Reply:
Well, the target is mkdocs, so not really an issue. :)